client/dist/style.css

```diff
@@ -132,6 +152,124 @@
   bottom: 15px;
 }
 
+/* Product Info Styles  =====================================================*/
+.styles-thumb {
+  height: auto;
+  width: 55px;
+  overflow: hidden;
+  margin: 0 auto;
+}
+
+.style-title {
+  font-size: larger;
+  margin-top: 20px;
+  margin-bottom: 10px;
+}
+
+.style-title, .product-info-align {
+  margin-left: 8px;
+}
+
+.product-top {
+  margin-top: 50px;
+}
+
+.img-cont {
+  height: 55px;
+  width: 55px;
+  border-radius: 50%;
+  overflow: hidden;
+  background-color: black;
+  margin: 8px;
+}
+
+.img-cont.active {
+  box-shadow: 5px 5px 5px cyan;
+}
+
+.red {
+  color: red;
+  text-decoration: line-through;
+}
+
+.new-price {
+  color: black;
+  text-decoration: none;
+}
+
+.badge {
+  position: absolute;
+  z-index: 5;
+}
+
+.read-all-reviews {
+  font-size: small;
+  text-decoration: underline;
+  padding-bottom: 20px;
+}
+
+.read-all-reviews:hover {
+  color: red;
+}
+
+.product-buttons {
+  display: inline-block;
+  margin: 10px;
+}
+
+.add-to-cart {
+  width: 63%;
+}
+
+.button-text {
+  margin-right: 90px;
+}
+
+.size-but {
+  min-width: 35%;
+}
+
+.feature-div {
+  /* border: none; */
+  border-width: 1px;
+  border-style: solid;
+  border-right: rgba(128, 128, 128, 0.419);
+  border-top: rgba(128, 128, 128, 0.419);
+  border-bottom: rgba(128, 128, 128, 0.419);
+}
+
+.feature-span {
+  line-height: 30px;
+}
+
+/* Font Awesome Styles  =====================================================*/
+
+.fa {
```

Comment:
Not sure if you have already, But I believe I remember Cindy saying shes using Font Awesome so make sure your styling is not in conflict with hers?

Reply:
I added a fa-pd (for product details) class which styles my font-awesome icons so @cindyryoo7 won't have overlap with me.